src/components/Editor/Editor.js

```diff
@@ -2,19 +2,19 @@ import React from 'react';
 
 import './Editor.css';
 
-export default function Editor() {
+export default function Editor({ setTitle, setSubtitle, setFont, setAlign, setText }) {
   return (
     <div className="editor">
       <div className="form-control">
-        <input name="title" type="text" />
+        <input name="title" type="text" onChange={(e) => setTitle(e.target.value)} />
         <label htmlFor="title">Title</label>
       </div>
       <div className="form-control">
-        <input type="text" />
+        <input type="text" onChange={(e) => setSubtitle(e.target.value)} />
         <label>Subtitle</label>
       </div>
       <div className="form-control">
-        <select>
+        <select onChange={(e) => { setFont(e.target.value); }}>
```

Comment:
Here we can use onChange{(e) => setFont(e.target.value)} to get rid of the extra curly brackets. Just an FYI